src/routes/jira/security/workspaces/containers/jira-security-workspaces-containers-router.ts

```diff
@@ -2,8 +2,12 @@ import { Router } from "express";
 import {
 	JiraSecurityWorkspacesContainersPost
 } from "~/src/routes/jira/security/workspaces/containers/jira-security-workspaces-containers-post";
+import { JiraSecurityWorkspacesContainersSearchGet } from "./jira-security-workspaces-containers-search-get";
 
 export const JiraSecurityWorkspacesContainersRouter = Router();
 
-JiraSecurityWorkspacesContainersRouter.route("/search")
+JiraSecurityWorkspacesContainersRouter.route("/containers")
 	.post(JiraSecurityWorkspacesContainersPost);
+
+JiraSecurityWorkspacesContainersRouter.route("/containers/search")
+	.get(JiraSecurityWorkspacesContainersSearchGet);
```

Comment:
/nit
Do you think we should do POST instead of GET?
1. Other similar PR use POST https://github.com/atlassian/github-for-jira/pull/2250/files#diff-03703ee06cc033d8f443c9d8ba6116a55e2463b2976ff779f6976a5a23fd6213R82
2. GET means return same thing as long as param is same. But search obviously can change base on DB state.

Reply:
@gxueatlassian , these actions are governed by the connect [security module](https://developer.atlassian.com/cloud/jira/software/modules/security-information/). As per specs, `searchContainers` should be a `GET` request.